ksqldb-engine/src/main/java/io/confluent/ksql/services/SandboxedServiceContext.java

```diff
@@ -46,29 +48,33 @@ public static SandboxedServiceContext create(final ServiceContext serviceContext
     final SchemaRegistryClient schemaRegistryClient =
         SandboxedSchemaRegistryClient.createProxy(serviceContext.getSchemaRegistryClient());
     final ConnectClient connectClient = SandboxConnectClient.createProxy();
+    final Admin adminClient = serviceContext.getAdminClient();
```

Comment:
we should not be passing the real admin client to the sandbox - otherwise it becomes possible for the sandbox to make real changes to the kafka cluster. I recommend instead following the pattern of the above Sandboxes and create a proxy that proxies the admin client for read-only commands. 

Better yet, I think it might make sense to bake this into the `KafkaTopicClient`, which already has an api for `describeTopic`

Reply:
fixed.